src/components/interface/EcosystemHeader/EcosystemHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import { SystemStyleObject, Theme, Box } from "@mui/system";
+import { Panel, Typography } from "src/components/atomic";
+
+export type EcosystemHeaderProps = {
+    lpOptimizerTag: string;
+    lpOptimizerCount: number;
+    alphaVaultTag: string;
+    alphaVaultCount: number;
+};
+
+const EcosystemHeader = ({lpOptimizerTag, lpOptimizerCount, alphaVaultTag, alphaVaultCount}: EcosystemHeaderProps) => {
+  
+    const titleStyles: SystemStyleObject<Theme> = { 
+        fontSize: '40px', 
+        lineHeight: '1.2', 
+        color: '#E5E1F9',
+        fontWeight: '700',
+    };
+
+    const copyStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color: '#9B97AD', 
+        marginTop: "8px",
+    }
+    
+    const tagStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+    };
+
+    const boxStyles: SystemStyleObject<Theme> = {
+        display: "flex",
+        borderRadius: "4px",
+        padding: "0px 8px",
+    }
+      
+    const renderContent = () => {
+        return (
+            <Panel sx={{padding: 0, background: 'transparent'}}>
+                <Box>
+                    <Typography variant='h1' sx={titleStyles}>
+                        VOLTZ ECOSYSTEM
+                    </Typography>
+                    <Typography variant='body1' sx={copyStyles}>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed nec sit iaculis cras elit dictum massa. Sit metus amet, tincidunt odio. Tristique sagittis, nisl in eu eu vestibulum et. Ut sed mauris urna justo, dictumst molestie posuere.
```

Comment:
Can you update with proper copy here?

Reply:
Let's add the copies during the QA process so Simon/Artur/Gerard can see where this is placed.

Reply:
Agreed! Please make a note for the copy owners!